docs/TESTING.md

```diff
@@ -1,53 +1,59 @@
 ### Test instructions (for developers):
 
-This is a rough sketch, some more background is found in [JM wiki](https://github.com/Joinmarket-Org/joinmarket/wiki/Testing)
+Work in your `jmvenv` virtualenv as for all Joinmarket work. Make sure to have [bitcoind](https://bitcoin.org/en/full-node) 0.17 or newer installed. Also need miniircd installed to the root (i.e. in your `joinmarket-clientserver` directory):
 
-Make sure to have [bitcoind](https://bitcoin.org/en/full-node) 0.17 or newer installed. Also need miniircd installed to the root (i.e. in your `joinmarket-clientserver` directory):
+    (jmvenv)$ cd /path/to/joinmarket-clientserver
+    (jmvenv)$ git clone https://github.com/Joinmarket-Org/miniircd
 
-    cd ~/joinmarket-clientserver
-    git clone https://github.com/Joinmarket-Org/miniircd
+Install the test requirements:
 
-Install the test requirements (still in your virtualenv as mentioned above):
+    (jmvenv)$ pip install -r requirements/testing.txt
 
-    pip install -r requirements/testing.txt
+#### Running the test suite.
 
-Running the test suite should be done something like (advisable to wipe ~/.bitcoin/regtest first):
+Have a `bitcoin.conf` ready in some location, whose contents only need to be:
 
-    pytest --btcconf=/path/to/bitcoin.conf --btcroot=/path/to/bitcoin/bin/ --btcpwd=whatever --nirc=2 --ignore test/test_full_coinjoin.py -p no:warnings
-    
-(you'll first want to copy the regtest_joinmarket.cfg file from the test/ directory to the root directory,
-this file will need minor edits for your btc configuration).
+```
+rpcuser=bitcoinrpc
+rpcpassword=123456abcdef
```

Comment:
Add `fallbackfee=0.0002` here. See #609

Reply:
Yes, thanks for the catch. Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/664/commits/9638a16158671f229ef4c8f89f9542cd24d0dd28